docs/functions.rst

```diff
@@ -33,6 +33,10 @@ Then it goes through **all** other needs and checks if the value of their ``sour
 the ``target_option``.
 If this is the case, their IDs get stored and finally returned.
 
+In case the ``source_option`` contains a ``*`` (e.g. ``sphinxcontrib.test_reports.*``), the function will check if the
+``target_option`` matches the ``source_option``. This is useful if one wants to link multiple test results to one test case.
+
```

Comment:
I'm not sure, if this documentation is correct.
`target_option` is now used as regex-pattern in all cases and `sphinxcontrib.test_reports.*` works, because of `.*`, which allows everyhing else after the given string, so also ´sphinxcontrib.test_reportsTHIS_IS_NEW`.

It's not only about the `*`, but the `.*`.

Ohh wait, this does not even work with `sphinxcontrib.test_reports.*`, as `.` is a special character. `.` must get escaped: `sphinxcontrib\.test_reports\..*` .

For some examples see: https://regex101.com/r/XTnHX6/1
![image](https://github.com/useblocks/sphinx-test-reports/assets/998700/a676639c-8682-4b60-91b1-571f9363b1bc)

Such complex `target_option` values are not covered by the test cases, as there simpler targets like `b` are used.